cray_product_catalog/query.py

```diff
@@ -170,6 +177,28 @@ def get_product(self, name, version=None):
         return matching_products[0]
 
 
+def loadConfigMapData(configmaps):
+    config_map_data = {}
+    for cm in configmaps:
+        if (not cm.metadata.name.startswith(PRODUCT_CATALOG_CONFIG_MAP_NAME)):
+            continue
+        for product_name, product_versions in cm.data.items():
+            for product_version, product_version_data in safe_load(product_versions).items():
+                cm_key = product_name + ':' + product_version
+                if (cm_key in config_map_data):
+                    config_map_data[cm_key] = merge_dict(config_map_data[cm_key], product_version_data)
+                else:
+                    config_map_data[cm_key] = product_version_data
+
+    products = []
+
+    for key, product_version_data in config_map_data.items():
+        product_name, product_version = key.split(':')
```

Comment:
If product version or name has `:` , it will be an issue here.
Can we make a dictionary of dictionary e.g.
cm_data = {product_name: {prod_version1: prod_data, prod_verison2: data, .... }}

When we want to get the product version, just cm_data